mobile/android/app/build.gradle

```diff
@@ -128,7 +128,7 @@ android {
     }
 
     defaultConfig {
-        applicationId "com.origincatcher"
+        applicationId "com.originmarketplace"
```

Comment:
The applicationId has to remain as `com.origincatcher` because that is what it was originally published to the Play store as.

Reply:
This is what I was having issues with before as well.

Reply:
Was having issues when running from CLI. Will revert it if it works fine otherwise.

Reply:
What were you running? You might need to use the android command as defined in `package.json`.

Reply:
@tomlinton Okay, just checked it. Adding the `appId` param works. Have reverted the change. Thanks.